---
layout: page
permalink: /research/
title: Research
tagline: Michael G. Harvey
tags: [about, research, Michael G. Harvey]
modified: 5-14-19
comments: true
image:
  feature: siberia.jpg
---

***

I study mechanisms responsible for broad patterns of organismal diversity using comparative studies at the population level. My work spans the fields of evolutionary genomics, population genetics, and macroevolution. Current and ongoing research projects include:
<br><br>

## Comparative research on avian speciation
***

![structure!](/images/structure.png)
<br><br>
**Main collaborators:** <a href="https://briantilstonsmith.wordpress.com/" target="_blank">Brian Tilston Smith</a> (AMNH), Glenn Seeholzer (AMNH), Haw Chuan Lim (USNM), Alex Aleixo (MPEG), Camila Ribas (INPA), <a href="https://sites01.lsu.edu/faculty/brumfieldlab/" target="_blank">Robb Brumfield</a> (LSU)
<br><br>
Speciation is well characterized in a few systems, mostly in the Temperate Zone. However, speciation may progress quite differently in lineages in the world's tropical regions, the engines of most new species diversity. I use population genomic, phenotypic, behavioral, and ecological data to characterize speciation in tropical bird lineages. I also conduct comparative studies across regions to assess how the process of speciation varies in distinct geographic or taxonomic contexts. This research is critical to evaluate whether current understanding of speciation is of general importance. 
<br><br>
**Papers:** <a href="http://mgharvey.github.io/docs/Harveyetal2017b.pdf" target="_blank">Harvey et al. 2017</a>, <a href="http://mgharvey.github.io/docs/Limetal2017.pdf" target="_blank">Lim et al. 2017</a>, <a href="http://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.2001073" target="_blank">Smith et al. 2017</a>, <a href="http://mgharvey.github.io/docs/Smithetal2014b.pdf" target="_blank">Smith et al. 2014</a>
<br><br>

## Macroevolutionary impacts of population-level processes
***

![bamm!](/images/bamm.png)
<br><br>
**Main collaborators:** <a href="https://briantilstonsmith.wordpress.com/" target="_blank">Brian Tilston Smith</a> (AMNH), Glenn Seeholzer (AMNH), <a href="http://www.raboskylab.org/" target="_blank">Dan Rabosky</a> (UMich), <a href="https://sites01.lsu.edu/faculty/brumfieldlab/" target="_blank">Robb Brumfield</a> (LSU)
<br><br>
Although diversity within species varies widely, this variation may or may not persist over long evolutionary timescales to impact trends in species diversity. I combine population genetic data on intraspecific diversity with phylogenetic data on species diversification through time to assess the links between diversity across timescales. If population-level patterns persist to macroevolutionary timescales, this supports the general evolutionary importance of processes at work within species. Alternatively, population-level processes may be ephemeral and secondary to other limits on species proliferation.
<br><br>
**Papers:** Harvey et al. In press, <a href="http://mgharvey.github.io/docs/Harveyetal2017.pdf" target="_blank">Harvey et al. 2017</a>
<br><br>

## Applications of genomic data to comparative studies of non-model organisms
***

![genomics!](/images/genomics.png)
<br><br>
**Main collaborators:** <a href="http://faircloth-lab.org/" target="_blank">Brant Faircloth</a> (LSU), <a href="http://baddna.org/Welcome.html" target="_blank">Travis Glenn</a> (UGA), <a href="https://briantilstonsmith.wordpress.com/" target="_blank">Brian Tilston Smith</a> (AMNH), <a href="https://sites.oxy.edu/mccormack/" target="_blank">John McCormack</a> (Occidental), <a href="https://sites01.lsu.edu/faculty/brumfieldlab/" target="_blank">Robb Brumfield</a> (LSU)
<br><br>
New DNA sequencing technologies are able to produce massive amounts of genetic data, but their application to research in non-model organisms is in its infancy. I have been working to develop and test approaches for obtaining data appropriate for comparative evolutionary studies. We developed a method involving sequence capture of markers linked to highly conserved regions of the genome (<a href="http://www.ultraconserved.org" target="_blank">Ultraconserved elements</a>). I have also been working with restriction digest associated DNA sequencing (RAD-Seq) and whole-genome resequencing. I have developed laboratory protocols, bioinformatics pipelines, and analytical tools that allow these powerful genomic techniques to be brought to bear on non-model species and samples from museum collections.
<br><br>
**Papers:** <a href="http://mgharvey.github.io/docs/Harveyetal2016.pdf" target="_blank">Harvey et al. 2016</a>, <a href="http://mgharvey.github.io/docs/Smithetal2014.pdf" target="_blank">Smith et al. 2014</a>, <a href="http://mgharvey.github.io/docs/Fairclothetal2012.pdf" target="_blank">Faircloth et al. 2012</a>
<br><br>

## Avian plumage evolution within and among species
***

![plumage!](/images/photoscan.png)
<br><br>
**Main collaborators:** <a href="http://www.raboskylab.org/" target="_blank">Dan Rabosky</a> (UMich)
<br><br>
Birds have varied and spectacular plumages, but the source of this diversity is still largely unknown. Do plumage differences between species originate from variation within species? At what rates do different plumage characters, such as color or pattern, change through evolutionary time? In order to address these questions, I am developing high-throughput methods to quantify plumage from museum specimens using full-spectrum digital imagery, photogrammetry, and computer vision and machine learning. 

